File: app/src/main/java/com/example/marcin/meetfriends/ui/my_schedule/planned_events/PlannedEventsModule.java
package com.example.marcin.meetfriends.ui.my_schedule.planned_events;

import dagger.Binds;
import dagger.Module;

/**
 * Created by marci on 2018-01-04.
 */

@Module
public abstract class PlannedEventsModule {

  @Binds
  abstract PlannedEventsContract.View bindsView(PlannedEventsFragment view);

  @Binds
  abstract PlannedEventsContract.Presenter bindPresenter(PlannedEventsPresenter presenter);
}

File: app/src/main/java/com/example/marcin/meetfriends/ui/my_schedule/planned_event/PlannedEventsModule.kt
package com.example.marcin.meetfriends.ui.my_schedule.planned_event

import com.example.marcin.meetfriends.ui.my_schedule.confirmed_events.PlannedEventsContract
import com.example.marcin.meetfriends.ui.my_schedule.confirmed_events.PlannedEventsPresenter
import dagger.Binds
import dagger.Module

/**
 * Created by MARCIN on 2017-11-13.
 */
@Module
abstract class PlannedEventsModule {

  @Binds
  abstract fun bindView(view: PlannedEventsFragment): PlannedEventsContract.View

  @Binds
  abstract fun bindPresenter(presenter: PlannedEventsPresenter): PlannedEventsContract.Presenter
}
